lib/node_modules/@stdlib/array/base/reshape/benchmark/benchmark.js

```diff
@@ -0,0 +1,206 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var isArray = require( '@stdlib/assert/is-array' );
+var onesnd = require( '@stdlib/array/base/onesnd' );
+var pkg = require( './../package.json' ).name;
+var reshape = require( './../lib' );
+
+
+// MAIN //
+
+bench( pkg+':ndims=2,size=100,lexicographic', function benchmark( b ) {
```

Comment:
Use prime factorization to ensure that these benchmarks all involve nested arrays having the same number of elements. This allows consumers to more readily infer expected perf as the number of dimensions increases.

```
Prime factorization of 100:

1, 2, 4, 5, 10, 20, 25, 50, 100

dims 2: 10, 10
dims 3: 4, 5, 5
dims 4: 5, 2, 5, 2
dims 5: 5, 2, 1, 5, 2
```